bin/grocery-delivery

```diff
@@ -220,14 +220,17 @@ def setup_config
 end
 
 def get_repo
-  repo_path = File.join(GroceryDelivery::Config.master_path,
-                        GroceryDelivery::Config.reponame)
-  r = BetweenMeals::Repo.get(GroceryDelivery::Config.vcs_type, repo_path,
-                             GroceryDelivery::Log)
-  if GroceryDelivery::Config.vcs_path
-    r.bin = GroceryDelivery::Config.vcs_path
```

Comment:
Yes - this is how people can override the path to git if they install it some place interesting... we used to use it when git was in /usr/local/fb-git/whateverthefuck.